app/assets/javascripts/dashboard.js

```diff
@@ -45,7 +45,13 @@
         $('.node-type-' + type).prop('checked', checked);
  
       }
-
+      $('.node-type-all').click(function () {
+        $('.node-type').prop('checked', this.checked);
+    });
+    $('.node-type').change(function () {
```

Comment:
This should be after the `.node-type`(in the same block). They are both related to checking/unchecking boxes functionality.

Reply:
@Keshavsethi kindly reply

Reply:
Sure, I will make changes.